www/include/options/accessLists/reloadACL/reloadACL.php

```diff
@@ -88,11 +99,16 @@
 $res = $pearDB->query("SELECT DISTINCT * FROM session");
 $session_data = array();
 $cpt = 0;
-$form = new HTML_QuickFormCustom('select_form', 'POST', "?p=".$p);
+$form = new HTML_QuickFormCustom('select_form', 'POST', "?p=" . $p);
 
-while ($r = $res->fetchRow()) {
-    $resUser = $pearDB->query("SELECT contact_name, contact_admin FROM contact WHERE contact_id = '".$r["user_id"]."'");
-    $rU = $resUser->fetchRow();
+while ($r = $res->fetch()) {
+    $resUser = $pearDB->prepare(
```

Comment:
I think you can place this subquery directly in the first request to relieve the database (SELECT DISTINCT * FROM session)

Reply:
no I can't as this query use the results of the previous one